### PR TITLE
Use LinkedIn API to get images for users

### PR DESCRIPTION
**Pull request by reshadbinharun**
* LinkedIn API (r_liteprofile permission) allows getting firstName, lastName, and profile picture

This PR sets up to get public link for LinkedIn profile photo URL and store it on the alumni/student record

### CODE REVIEW

File: client/src/components/Profile.js [hunk 1]
```diff
@@ -22,9 +23,7 @@ export default class Profile extends Component {
         const isViewOnly = this.props.isViewOnly;
 
         const linkedInUpdate = (
-            <Button primary>
-                Update info with LinkedIn
-            </Button>
+            <LinkedInUpdate/>
```
Comment (reshadbinharun) [inline client/src/components/Profile.js:26]: TODO: email should be passed in as prop here if we do in fact plan to use this method of fetching profile pics

File: client/src/components/Signup.js [hunk 2]
```diff
@@ -184,13 +184,17 @@ export default class Signup extends React.Component {
             e.preventDefault();
             const endPoint = this.props.isAlumni ? '/alumni' : '/student';
             try {
-                const result = makeCall(payload, endPoint, 'post')
+                const result = await makeCall(payload, endPoint, 'post')
```
Comment (reshadbinharun) [inline client/src/components/Signup.js:187]: Bonus fixes!

**Comment by reshadbinharun** [comment]
We should discuss if it's worth using the LinkedIn API at all!
Also (https://thenextweb.com/dd/2015/02/12/linkedin-takes-aim-developers-plans-lock-apis/)[Boo!]

File: client/src/components/Profile.js (hunk 1 above)
Comment (awilli70) [reply client/src/components/Profile.js:26]: Another option would be to just pass in the profile data to this component and store it as a state - we would still need to have isViewOnly

**Comment by awilli70** [comment]
LinkedIn at this point seems to have very limited utility for the project as a whole - if its sole purpose in the application is just to grab a profile picture url, it offers very little convenience for the increase in code and means we rely on an API that seems pretty likely to change given the amount of features they have already closed off

**Comment by reshadbinharun** [comment]
> LinkedIn at this point seems to have very limited utility for the project as a whole - if its sole purpose in the application is just to grab a profile picture url, it offers very little convenience for the increase in code and means we rely on an API that seems pretty likely to change given the amount of features they have already closed off
> 
> We can talk more about this on Saturday!

Agreed! We can limit our potential use to only grabbing the profile photo, and have this end-point available should we choose to use it!

**Pull request merged**